classes/passwd.oeclass

```diff
@@ -19,18 +19,14 @@ GROUP_FILES ?= "${SRCDIR}/group"
 PASSWD_POSTFUNC:machine = " do_install_passwd"
 do_install[postfuncs] += "${PASSWD_POSTFUNC}"
 do_install_passwd () {
-	i=0
 	for f in ${PASSWD_FILES} ; do
-		mkdir -p ${D}/${passwddir}
-		let i=$i+1
-		cp $f ${D}/${passwddir}/${PN}.$i
+		mkdir -p ${D}/${passwddir}/${PN}
+		cp $f ${D}/${passwddir}/${PN}/
```

Comment:
2 counter arguments to that:

1. That would be different from most other similar interfaces, thus violating principle of least surprise.
2. That would prevent the use of files from other locations, such as ${B} and ${STAGE_DIR}.